isso/views/comments.py

```diff
@@ -1141,6 +1142,12 @@ def login(self, env, req):
             isso_host_script = self.isso.conf.get("server", "public-endpoint") or local.host
             return render_template('login.html', isso_host_script=isso_host_script)
 
+    def logout(self, env, req):
+        response = redirect("/admin")
+        response.headers.add("Set-Cookie", "admin-session=none; path=/; max-age=0;")
+        response.headers.add("X-Set-Cookie", "isso-admin-session=none; path=/; max-age=0;")
```

Comment:
I can't remember why initially this whole `X-Set-Cookie`/`Set-Cookie` duality started...
Probably something related to proxies stripping certain headers? Or are we just cargo-culting/copy-pasting this everywhere for no good reason?

Reply:
Ah, to get around cross-origin restrictions, see https://github.com/posativ/isso/commit/6691810316ef1fc445251c1a1f3e02dbcdc54df6